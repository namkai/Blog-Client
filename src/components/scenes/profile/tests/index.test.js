import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import { Profile } from '../index';

it('renders correctly', () => {
	const profile = shallow(
		<Profile />
	);
	expect(toJson(profile)).toMatchSnapshot();
});