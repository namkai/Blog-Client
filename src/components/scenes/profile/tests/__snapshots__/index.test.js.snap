// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Component
  backgroundPhoto={[Function]}
  name={[Function]}
  profilePhoto={[Function]}
/>
`;
